### PR TITLE
New JsonSchemas for Label descriptors

### PR DESCRIPTION
**Pull request by veramach-adobe**
Json schemas to support the new label descriptors

@cmathis @prabhum2

### CODE REVIEW

File: schemas/descriptors/label/descriptorLabel.schema.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/schemadescriptor#/definitions/descriptor"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/labeldescriptor#/definitions/labeldescriptor"
```
Comment (cmathis) [inline schemas/descriptors/label/descriptorLabel.schema.json:35]: No reason to create this external labeldescriptor file. Just add the "xdm:labels" field definition in the local definition here.

File: schemas/descriptors/label/labeldescriptor.schema.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/labeldescriptor",
```
Comment (cmathis) [inline schemas/descriptors/label/labeldescriptor.schema.json:8]: No need for this file. Simply add the "xdm:labels" field definition to the new https://ns.adobe.com/xdm/common/descriptors/descriptorLabel

File: schemas/descriptors/label/descriptorLabel.schema.json (hunk 1 above)
Comment (veramach-adobe) [reply schemas/descriptors/label/descriptorLabel.schema.json]: Yes. My bad. There is no reason for that file.

**Comment by veramach-adobe** [comment]
@cmathis Updated with the latest commit

File: schemas/descriptors/label/descriptorLabel.schema.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+              },
+              "labelValue": {
+                "type": "string"
+              }
+            }
+          },
+          "meta:xdmType": "array"
+        }
+      },
+      "oneOf" : [
```
Comment (cmathis) [inline schemas/descriptors/label/descriptorLabel.schema.json:53]: What is the purpose of the "oneOf" here? Are we saying this descriptor can have have either a xdm:labels OR a xdm:sourceProperty? 

Seems like you want both of those to always be required so simply do:
"required": ["xdm:sourceProperty", "xdm:labels"]

Comment (veramach-adobe) [reply schemas/descriptors/label/descriptorLabel.schema.json]: Yes, Taken care of. I re-used the relationship descriptor schema definitions that defines this way. I should given a closer look.

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**